assets/hip-206/solidity/IHederaTokenService.sol

```diff
@@ -133,6 +275,44 @@ interface IHederaTokenService {
     /// @param token The token to be associated with the provided account
     function dissociateToken(address account, address token) external returns (int responseCode);
 
+    /// Creates a Fungible Token with the specified properties
+    /// @param token the basic properties of the token being created
+    /// @param initialTotalSupply Specifies the initial supply of tokens to be put in circulation. The
+    /// initial supply is sent to the Treasury Account. The supply is in the lowest denomination possible.
+    /// @param decimals the number of decimal places a token is divisible by.
+    /// @param expiry expiry properties of a Hedera token - second, autoRenewAccount, autoRenewPeriod
+    /// @param fixedFees list of fixed fees to apply to the token
+    /// @param fractionalFees list of fractional fees to apply to the token
+    /// @param keys list of keys to set to the token
+    /// @return bool whether the create was successful
+    /// @return address the address of the created token
+    function createFungibleToken(
+        HederaToken memory token, 
+        uint initialTotalSupply, 
+        uint decimals, 
+        Expiry memory expiry, 
+        FixedFee[] memory fixedFees, 
+        FractionalFee[] memory fractionalFees, 
+        Key[] memory keys) external returns (bool success, bytes tokenAddress);
```

Comment:
obsessive nit: I would put `keys` earlier in the list because they are more important.  Before the custom fees.  Possibly just before expiry.  Custom fees may be empty and I would want mandatory fields before optional.